### PR TITLE
Test fixlets

### PR DESCRIPTION
**Pull request by fbuihuu**

### CODE REVIEW

**Review by bluca — APPROVED** [review]

**Comment by bluca** [comment]
```
15:06:17 make: Entering directory '/root/systemd/test/TEST-01-BASIC'
15:06:18 mktemp: failed to create directory via template ‘/var/tmp/systemd-tests/systemd-test.XXXXXX’: No such file or directory
```

**Review by yuwata — APPROVED** [review]

File: test/test-functions [hunk 1]
```diff
@@ -3036,6 +3036,7 @@ _test_cleanup() {
         fi
         [[ -n "$TESTDIR" ]] && rm -vfr "$TESTDIR"
         [[ -n "$STATEFILE" ]] && rm -vf "$STATEFILE"
+        [[ -n "$STATEDIR" ]] && rm -vfr "$STATEDIR"
```
Comment (bluca) [inline test/test-functions:3039]: This always removes TESTLOG, even on failure, so it cannot be consulted anymore. I'll change it to cleanup this only on success.

Comment (bluca) [reply test/test-functions:3039]: https://github.com/systemd/systemd/pull/27045

Comment (fbuihuu) [reply test/test-functions:3039]: Indeed, thanks for the follow-up.

**Pull request merged**